### PR TITLE
[MOD-2241] Support "statuschange" event on LikeButton

### PR DESCRIPTION
**Pull request by hansemannn**
JIRA: https://jira.appcelerator.org/browse/MOD-2241

### CODE REVIEW

File: ios/Classes/FacebookLikeButton.m [hunk 1]
```diff
@@ -20,11 +21,19 @@ -(FBSDKLikeControl*)like
 {
     if (like == nil) {
         like = [[FBSDKLikeControl alloc] init];
+
```
Comment (AngelkPetkov) [inline ios/Classes/FacebookLikeButton.m:24]: Remove this space please.

File: apidoc/LikeButton.yml [hunk 2]
```diff
@@ -148,3 +148,10 @@ properties:
         Set to `true` for sound.
     platforms: [iphone,ipad]
     type: Boolean
+
+events:
+
+  - name: statuschange
+    summary: | 
+        Fired when the like button status changed. To see if this change was triggered
```
Comment (AngelkPetkov) [inline apidoc/LikeButton.yml:156]: change to 'status has changed'

File: ios/Classes/FacebookLikeButton.m (hunk 1 above)
Comment (hansemannn) [inline ios/Classes/FacebookLikeButton.m:24]: What, why? I added it on purpose to separate the different semantical actions.

Comment (AngelkPetkov) [inline ios/Classes/FacebookLikeButton.m:24]: You're assigning like on line 24 then you break a line and use it. I think it looks better if you just have it a 3 line code block. But up to you, you don't have to change it.

**Comment by AngelkPetkov** [comment]
FT passed for IOS, currently setting up the android environment to build/compile for android.

**Comment by ashcoding** [comment]
I'll help take a look into building for Android.

**Comment by ashcoding** [comment]
[facebook-android-5.1.0.zip](https://github.com/appcelerator-modules/ti.facebook/files/237935/facebook-android-5.1.0.zip)
Compiled PR #47 into the above android module for you to test.

**Pull request closed without merging**